src/DFC_BusinessOntology.rdf

```diff
@@ -37,7 +37,7 @@
         <dc:title xml:lang="en">Data Food Consortium Business</dc:title>
         <vann:preferredNamespacePrefix>dfc-b</vann:preferredNamespacePrefix>
         <rdfs:comment xml:lang="en">A common vocabulary for digital food platforms (Business Part)</rdfs:comment>
-        <owl:versionInfo>1.12.0</owl:versionInfo>
```

Comment:
Noting you've tagged this v1.14, do we need a release (also would it be v1.14 or v1.13.1)?

Reply:
Agreed this will form part of v.1.14 release, with Business Definitions PR.

Reply:
@Alcoz - you've reverted the v.1.14 flag in your last commit, was that intentional ? 😕